### DISCUSSION TITLE
509报错 帮助手册无对应内容

### DISCUSSION THREAD

**Discussion by SaidRain** [natfrp/wiki#303] [水深火热]
具体报错内容如下
[53/319394/731a] panic error: x509: invalid RDNSequence
[53/319394/731a] goroutine 121 [running]:

**Comment by SaidRain** [comment]
另外 我是按照教程想弄远程管理的；成都4和成都7都试过了，皆报错

**Comment by fakeboboliu** [comment]
您好， 此问题应当是以下两种情况同时出现导致的：

 - 您的网络 TLS 流量被劫持导致自检功能被关闭，这是在 `v0.45.0-sakura-3` 添加的用于用户电脑被木马感染又不愿杀毒时的兼容方案
 - 您的系统证书出现损坏或配置错误的自签名证书

我们建议您考虑检查系统环境，或者使用历史版本，如 [0.45.0-sakura-2.3](https://nyat-static.globalslb.net/natfrp/client/0.45.0-sakura-2.3/)，此版本会总是使用内置证书检查，从而暴露问题。

**Comment by fakeboboliu** [comment]
在一位信息提供较全面的用户帮助下，我们发现了在使用 启动器 时启用 自动HTTPS 功能会出现此问题的具体场景，并作出了修复。

如果这恰好是您的情况，可以将 [此压缩包](https://github.com/natfrp/wiki/files/11161351/frpcfix-23040601.zip) 覆盖到 `C:\Program Files\SakuraFrpLauncher` 中，即可修复，我们会在今天(UTC+8)晚些时候推送此更新。

最后，感谢您的反馈。
